path.go

```diff
@@ -528,8 +528,22 @@ func (n *selectorNode) filter(node ast.Node) (ast.Node, error) {
 			}
 			return filtered, nil
 		}
+	case ast.AnchorType:
+		for _, value := range node.(*ast.AnchorNode).Value.(*ast.MappingNode).Values {
```

Comment:
The `Value` of `AnchorNode` is not always `MappingNode`. Therefore, this type assertion may cause panic.

Reply:
Fixed this by calling the filter function with `node.(*ast.AnchorNode).Value`